### DISCUSSION TITLE
How to monitor if HA is connected to Homematic and send notification if not ?

### DISCUSSION THREAD

**Discussion by pixeldoc2000** [SukramJ/aiohomematic#1572] [Q&A]
What is the best Method to check if HA is still connected to Homematic (raspberrymatic)?

I have tried various things, but did not find a good solution like this Template to check Status of RMUSB connected to raspberrymatic: `{{ now() - states.sensor.hmip_rfusb_00041d89b55xxx_carrier_sense_level.last_changed > timedelta(hours=1) }}`

I try to detect if HA did not fully reconnect to Homematic after a HA restart / update for example. Sometime I forget to check if the connection is reestablished, otherwise my automations are broken and wife is mad next day.

I would like to achieve something similar like this: Zigbee2Mqtt has `binary_sensor.zigbee2mqtt_bridge_connection_state`, if this change to offline, I send myself a notification and display Warning in UI.

Maybe something like "Bridge Online Status" for hahomematic would be worth implementing?

**Comment by SukramJ** [comment]
> I try to detect if HA did not fully reconnect to Homematic after a HA restart / update for example. Sometime I forget to check if the connection is reestablished,

The integration uses an automatically reconnect (could take some minutes).


To check an individual entity is available you could try (I did not test it):

`states.sensor.hmip_rfusb_00041d89b55xxx_carrier_sense_level.state != 'unavailable'`
